src/modules/layoutUiRules/index.js

```diff
@@ -4,7 +4,14 @@ export {
     getAxisMaxNumberOfItemsByVisType as getAxisMaxNumberOfItems,
     getAxisMaxNumberOfDimsByVisType as getAxisMaxNumberOfDimensions,
     getAxisMinNumberOfDimsByVisType as getAxisMinNumberOfDimensions,
-    hasAxisTooManyItemsByVisType as hasAxisTooManyItems,
     getAxisPerLockedDimByVisType as getAxisPerLockedDimension,
     getAllLockedDimIdsByVisType as getAllLockedDimensionIds,
 } from './rulesHelper'
+
+export {
+    hasAxisTooManyItemsByVisType as hasAxisTooManyItems,
+    isDimensionLocked,
```

Comment:
Technically, to follow the pattern, the internal names for `isDimensionLocked`, `isAxisFull` and `canDimensionBeAddedToAxis` should be suffixed by `...ByVisType`

Reply:
Fixed